packages/mobile/src/transactions/saga.ts

```diff
@@ -54,18 +55,31 @@ export function* sendAndMonitorTransaction<T>(
     yield call(wrapSendTransactionWithRetry, txId, sendTxMethod)
     yield put(transactionConfirmed(txId))
 
+    txTime = Date.now() - txTime
     if (currency === CURRENCY_ENUM.GOLD) {
+      Logger.debug(
+        TAG + '@sendAndMonitorTransaction',
+        `send_gold_transaction_confirmed Performance (ms) = ` + txTime.toString()
+      )
+      CeloAnalytics.track(CustomEventNames.send_gold_transaction_confirmed, { txTime })
```

Comment:
These seem redundant with the analytics tracking above. How about we just make currency a property of the `CustomEventNames.send_transaction_confirmed` event?

Reply:
Okey, got it.